test/Amazon.AspNetCore.DataProtection.SSM.Tests/SSMXmlRepositoryTests.cs

```diff
@@ -631,6 +632,104 @@ public void StorageModeIntelligentTiering()
             repository.StoreElement(key, null);
         }
 
+#if NET9_0_OR_GREATER
+        [Fact]
```

Comment:
code looks good. can you add one test case for verifying the `                    // Stop processing deletions to avoid deleting a revocation entry for a key that we failed to delete.` scenario

Reply:
Done. Please review.